server/routes.js

```diff
@@ -99,9 +99,19 @@ module.exports = (expressApp, nextApp) => {
       additionalLink = `Additional Link: <a href="${bodyLink}">${bodyLink}</a></br>`;
     }
 
+    let relatedCollectives = 'Related Collectives: ';
+    if (body.relatedCollectives?.length > 0) {
+      for (let i = 0; i < Math.min(body.relatedCollectives.length, 50); i++) {
+        relatedCollectives = `${relatedCollectives}<a href='${body.relatedCollectives[i]}'>${body.relatedCollectives[i]}</a>, `;
+      }
+    }
+
```

Comment:
:ice_cream: To prevent the leftover comma at the end

```suggestion
    let relatedCollectives = 'Related Collectives: ';
    if (body.relatedCollectives?.length > 0) {
      relatedCollectives = body.relatedCollectives
        .slice(0, 50)
        .map(url => `<a href='${url}'>${url}</a>`)
        .join(', ');
    }

```